patches/chrome-browser-resources-settings-privacy_page-privacy_page.html.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/resources/settings/privacy_page/privacy_page.html b/chrome/browser/resources/settings/privacy_page/privacy_page.html
+index c938b42ff11691e4e5e915e7c4d6f36de58d82cb..2dde6c32cf91eebd7084fe3c3e6470648bedd1a2 100644
+--- a/chrome/browser/resources/settings/privacy_page/privacy_page.html
++++ b/chrome/browser/resources/settings/privacy_page/privacy_page.html
+@@ -89,7 +89,7 @@
+         <settings-toggle-button pref="{{prefs.safebrowsing.enabled}}"
+             label="$i18n{safeBrowsingEnableProtection}">
+         </settings-toggle-button>
+-<if expr="_google_chrome">
++<if expr="_chromium">
```

Comment:
@simonhong Not sure if there's a better way to perform this check. It seems like "_brave_chrome" would be a good variable to have, but when I looked it seemed that a lot of our code is assuming that "not _google_chrome" == "brave".

Reply:
I also think it's good to add `_brave_chrome` that both means brave and official to grit defines list.

Reply:
actually for these files maybe it's better to set `_google_chrome` so we don't have to patch them. cc @yrliou

Reply:
in any case ping @yrliou because she dealt with some similar issues in a recent PR in the html pages